src/clojure_tictactoe/game/board.clj

```diff
@@ -0,0 +1,11 @@
+(ns clojure-tictactoe.game.board)
+
+(defn open?
```

Comment:
I think this name can be more explicit. What's "open"?

Reply:
It's now `space-is-open?`